docs/source/services/timelock_service/installation.rst

```diff
@@ -3,6 +3,16 @@
 Timelock Server Installation
 ============================
 
+This guide explains how to install the Timelock server, and to configure existing services to use Timelock Server rather than embedded AtlasDB time and lock services.
+The basic method is to first install Timelock without any clients, and then for each existing service, prepare the service and Timelock to talk to each other.
+
+Obtain the Timelock binaries
+----------------------------
+
+.. tip::
+
+   Internal consumers can skip these steps.
```

Comment:
Nice. I like this idea of reminding internal consumers where steps for internal deployment are different.